settings.sample.yaml

```diff
@@ -0,0 +1,16 @@
+# copy this file as `settings.local.yaml` and override any config provided under `conf/*.yaml`
+# or preferably use environment variables instead of this file
+# example:
+# `export SATQE_SERVER__HOSTNAME=myserver.redhat.com`
+---
+server:
```

Comment:
@JacobCallahan should keep an eye on this for #7997

Reply:
we can make this change later. current settings would likely work from robottelo.properties